drivers/xen/gntdev-dmabuf.c

```diff
@@ -198,7 +199,8 @@ static int dmabuf_exp_wait_released(struct gntdev_dmabuf_priv *priv, int fd,
 /* DMA buffer export support. */
 
 static struct sg_table *
-dmabuf_pages_to_sgt(struct page **pages, unsigned int nr_pages)
+dmabuf_pages_to_sgt(struct page **pages, unsigned int nr_pages,
+		    unsigned int data_ofs)
```

Comment:
Just to clarify: did you ran `checkpatch.pl` on this series? Formatting there looks odd.

Reply:
```
static struct sg_table *
dmabuf_pages_to_sgt(struct page **pages, unsigned int nr_pages,
		    unsigned int data_ofs)
```
don't know why, but in the code it is properly formatted:

![Screenshot from 2020-05-20 10-46-39](https://user-images.githubusercontent.com/827874/82419626-5aa13c00-9a87-11ea-89e6-c57d7832d770.png)